dash_docs/chapters/dash_core_components/Slider/index.jl

```diff
@@ -0,0 +1,41 @@
+module chapters_dash_core_components_slider
+
+using Dash, DashHtmlComponents, DashCoreComponents
+
+include("../../../utils.jl")
+
+export examples
+
+examples_path = joinpath(@__DIR__, "examples")
+
+slider1 = LoadExampleCode(string(examples_path, "/basic_slider.jl"))
```

Comment:
DRY or more specific naming (eg I think basic_slider and multi_slider would be preferable to slider1 and slider2)